libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/LoggerFrontend.kt

```diff
@@ -24,6 +27,13 @@ internal class MapboxCommonLoggerFrontend : LoggerFrontend {
         NativeLoggerWrapper.debug(message, NAV_SDK_CATEGORY)
     }
 
+    override fun logD(category: String?, lazyMsg: () -> String) {
+        when (LogConfiguration.getLoggingLevel()) {
```

Comment:
This level can change. We should keep in mind (better even in docs) that if you had level info, invoked the lazy log and then changed the level to debug, you won't see the log. I'm not sure how it works with usual logs now but in AS it's just a filter so you'll see the old logs as well.